corehq/apps/domain/models.py

```diff
@@ -164,6 +164,7 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
     goal_followup_rate = DecimalProperty()
     # intentionally different from and commtrack_enabled so that FMs can change
     commtrack_domain = BooleanProperty()
+    performance_threshold = IntegerProperty()
```

Comment:
i actually considered that and decided against since it would be much harder to change the default later